streamingpro-core/src/main/java/tech/mlsql/crawler/RestUtils.scala

```diff
@@ -0,0 +1,342 @@
+package tech.mlsql.crawler
+
+import net.csdn.common.path.Url
+import net.csdn.modules.transport.HttpTransportService.SResponse
+import net.csdn.modules.transport.{DefaultHttpTransportService, HttpTransportService}
+import org.apache.commons.lang3.exception.ExceptionUtils
+import org.apache.http.HttpResponse
+import org.apache.http.client.entity.UrlEncodedFormEntity
+import org.apache.http.client.fluent.{Form, Request}
+import org.apache.http.entity.ContentType
+import org.apache.http.entity.mime.{HttpMultipartMode, MultipartEntityBuilder}
+import org.apache.http.message.BasicNameValuePair
+import org.apache.http.util.EntityUtils
+import streaming.dsl.ScriptSQLExec
+import streaming.log.WowLog
+import tech.mlsql.common.JsonUtils
+import tech.mlsql.common.utils.distribute.socket.server.JavaUtils
+import tech.mlsql.common.utils.log.Logging
+import tech.mlsql.crawler.RestUtils.FunctionsUtils.executeWithRetrying
+import tech.mlsql.dsl.adaptor.DslTool
+import tech.mlsql.tool.{HDFSOperatorV2, Templates2}
+
+import java.nio.charset.Charset
+import scala.jdk.CollectionConverters.seqAsJavaListConverter
+import scala.util.control.Breaks.{break, breakable}
+
+object RestUtils extends Logging with WowLog {
+  def httpClientPost(urlString: String, data: Map[String, String]): HttpResponse = {
```

Comment:
Moved to core, it is convenient to use in projects such as streamingpro-mlsql and streamingpro-core.